example/types/ExampleContract.py

```diff
@@ -34,12 +34,10 @@
 from __future__ import annotations
 
 import copy
-from typing import Any, Iterable, NamedTuple, Sequence, Type, cast, overload
+from typing import Any, Iterable, NamedTuple, Optional, Sequence, Type, cast, overload
```

Comment:
why are we using Optional syntax instead of pipe? we recommend python 3.10 in the INSTALL.md file. I think it's 3.10+ when the recommended syntax changed to pipes